README.md

```diff
@@ -15,7 +15,6 @@ and ll of this are built in a beautiful and well-organized way!
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you made your commit messages capitalized to make your pull request professional and descriptive :blush: